src/main/java/forestry/arboriculture/tiles/TileLeaves.java

```diff
@@ -495,4 +499,19 @@ public boolean canNurse(IButterfly butterfly) {
 		ITree tree = getTree();
 		return !isDestroyed(tree, damage) && caterpillar == null;
 	}
+
+	@Override
+	public Biome getBiome() {
+		return worldObj.getBiome(pos);
+	}
+
+	@Override
+	public EnumTemperature getTemperature() {
+		return EnumTemperature.getFromValue(ForestryAPI.climateManager.getTemperature(worldObj, pos));
```

Comment:
Same problem as in DefaultClimateProvider 
Replace this line with `EnumTemperature.getFromBiome(Biome Biome, World world, BlockPos pos)` here or it can't detect `EnumTemperature.HELLISH`.